### ISSUE TITLE
Legend of Zelda (NES) thumbnail doesn't load for some reason

### ISSUE THREAD

**Issue by tlaanemaa** [Swordfish90/Lemuroid#185]
Hey 👋

I noticed that the cover thumbnail for Legend of Zelda on NES (crc32 `02bb0c56`) doesn't seem to load for some reason.
The libretro DB has an entry for that game at id `12524` (`Legend of Zelda, The (USA) (Rev A)`) thats according to [this](https://github.com/Swordfish90/Lemuroid/blob/336ad817c1ef548b624fad65c04c289336399c18/lemuroid-metadata-libretro-db/src/main/java/com/swordfish/lemuroid/metadata/libretrodb/LibretroDBMetadataProvider.kt#L152) should point to a thumbnail [here](http://thumbnails.libretro.com/Nintendo%20-%20Nintendo%20Entertainment%20System/Named_Boxarts/Legend%20of%20Zelda%2C%20The%20%28USA%29%20%28Rev%20A%29.png).

At first this file didn't exist so I added it in this [PR](https://github.com/libretro-thumbnails/Nintendo_-_Nintendo_Entertainment_System/pull/50) but this still did not fix the issue. The file exists on the page but the thumbnail still doesn't show up in my Lemuroid

Device: Android TV - Nvidia Shield PRO
Filename: Legend of Zelda, The (USA) (Rev A).nes
File crc32: 02bb0c56

Thanks in advance!

**Comment by Swordfish90** [comment]
Hi @tlaanemaa, thank you for doing all the required steps to get the thumbnail upstreamed. I have to say it worked, I have the same ROM and I can see the thumbnail now. I also checked in the local database on my phone and the cover URL is computed correctly.

Can you try scanning an empty directory and then rescanning your whole ROMs directory? What if you scan a directory that only contains that file?

**Comment by tlaanemaa** [comment]
Hey, no problem, I was trying to fix the problem the best I could.
I wasn't feeling like setting up an android dev env to dive into Lemuroid tho :D

I tried scanning like you said, that didn't work.
But then I just reinstalled the app and that worked. It seems there might be some sort of a cache? So before when the image didn't exist, it remembered that and didn't go looking again no matter what I did?

Also, it might be good (and I know this is probably way more difficult) to change the fallback logic so that it also checks for the existence of the image before falling back. This would avoid cases like I had where the crc32 matches but the image doesn't exist and would allow people to perhaps tweak the filename to get it to match instead.

Any case, the problems now solved for me.
Thanks for all the help!
